includes/ucf-news-feed.php

```diff
@@ -85,6 +85,26 @@ public static function get_news_items( $args ) {
 			return self::get_json_feed( $feed_url );
 		}
 
+		public static function get_external_stories( $args ) {
+			$params = array();
+
+			if ( isset( $args['limit'] ) ) {
+				$params['limit'] = $args['limit'];
+			}
+
+			if ( isset( $args['offset'] ) ) {
+				$params['offset'] = $args['offset'];
+			}
+
+			$param_string = urldecode( http_build_query( $params ) );
+
+			$feed_url = $args['feed_url'];
+
+			$url = "$feed_url?$param_string";
```

Comment:
This URL will be invalid if the feed url is modified in the plugin settings field to include extra query params.  Consider at least adding a note to the field description stating that query params aren't supported, or add logic here that separates existing query params in `$feed_url` and merges them with those in `$params`.

Reply:
Updated the message for this setting for the time being. We'll make a utility function in the future to handle params present in the URL.